### PR TITLE
feat(http): add scraper target swagger

### PR DESCRIPTION
**Pull request by kelwang**
Closes https://github.com/influxdata/platform/issues/2264

add swagger doc for frontend.

To start scrape a target, make a post request to create a new scraper target.
To stop scrape a target, make a delete request to the scraper target endpoint

###### Required for all non-trivial PRs
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: http/swagger.yml [hunk 1]
```diff
@@ -572,6 +572,96 @@ paths:
...
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ScraperTarget"
+      responses:
+        '200':
+          description: scraper target updated
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Macro"
```
Comment (goller) [inline http/swagger.yml]: This should be ScraperTarget

File: http/swagger.yml [hunk 2]
```diff
@@ -6195,6 +6285,28 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Source"
+    ScraperTarget:
+      type: object
+      properties:
+        id:
```
Comment (goller) [inline http/swagger.yml]: this should be a readonly field

File: http/swagger.yml [hunk 3]
```diff
@@ -6195,6 +6285,28 @@ components:
...
+          type: string
+        links:
```
Comment (goller) [inline http/swagger.yml]: This should refer to the `Links` schema

File: http/swagger.yml [hunk 4]
```diff
@@ -6195,6 +6285,28 @@ components:
...
+              type: object
+              properties:
+                self:
+                  type: string
+        name:
+          type: string
```
Comment (goller) [inline http/swagger.yml]: Add a description for name, name, url, orgID, and bucketID

File: http/swagger.yml [hunk 5]
```diff
@@ -6195,6 +6285,28 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Source"
+    ScraperTarget:
```
Comment (goller) [inline http/swagger.yml]: This should be ScraperTargetResponse;  Also, add a ScraperTargetRequest

File: http/swagger.yml [hunk 6]
```diff
@@ -6195,6 +6285,28 @@ components:
...
+        name:
+          type: string
+        type:
+          type: string
+          enum: [prometheus]
+        url:
+          type: string
+          example:  http://localhost:9090/metrics
+        orgID:
+          type: string
```
Comment (goller) [inline http/swagger.yml]: This Response object should also have org and bucket names.

Comment (goller) [reply http/swagger.yml]: As an aside, if an org or bucket no longer exists (deleted), do you check if they are gone?
If not, please add an issue.

File: http/swagger.yml [hunk 7]
```diff
@@ -572,6 +572,96 @@ paths:
...
               schema:
                 $ref: "#/components/schemas/Error"
+  /scrapertargets:
+    get:
+      tags:
+        - ScraperTargets
+      summary: get all scraper targets
+      responses:
+        '200':
+          description: all macros for an organization
```
Comment (desa) [inline http/swagger.yml]: `s/macros/scrape targets/`

File: http/swagger.yml [hunk 8]
```diff
@@ -6929,4 +7041,4 @@ components:
         properties:
           type: object
           description: Key/Value pairs associated with this label. Keys can be removed by sending an update with an empty value.
-          example: {"color": "ffb3b3", "description": "this is a description"}
+          example: {"color": "ffb3b3", "description": "this is a description"}
```
Comment (desa) [inline http/swagger.yml]: What's up with this change?

Comment (kelwang) [reply http/swagger.yml]: I think it's my vs code's formatter, kill an extra blank space at the end.

**Pull request merged**